features-json/webm.json

```diff
@@ -255,7 +255,7 @@
       "13":"a #4",
       "13.1":"a #4",
       "14":"a #4 #5 #6",
-      "14.1":"y",
+      "14.1":"a #4 #5 #6",
```

Comment:
![image](https://user-images.githubusercontent.com/1336881/116639168-d5b2e800-a9a2-11eb-8f48-2cc1bdfa3690.png)

Regarding webM: Safari 14.1 release notes on Catalina/Mojave suggests the feature is **not** dependent on the OS. I've been setting up my VM to test that got delayed due to work reasons. Let me get back to this soon, but in the mean time I don't mind keeping (reverting?) it partial.

Reply:
Yeah, I'm actually not sure why @Fyrd added 14.1 as supported in https://github.com/Fyrd/caniuse/commit/499c067dc51b8141840847cc52ef365d92c05a73#diff-7f7bcd7c4eeb5170b1aa0c458cdfbc6c9e2708e3daf5f3a6735baa8be1861123, given that TP is in the state you set in #5721, which is still true, isn't it? 
Or is fully supported now?
I can test 14.1 & TP on Catalina, but I'm quite sure that yesterday <https://tests.caniuse.com/webm> didn't work.

Reply:
On Catalina (10.15.7 (19H1030), Mid 2020 Intel MacBook Pro), neither in 14.1 nor TP any of <https://tests.caniuse.com/webm> work:

<img width="1436" alt="Bildschirmfoto 2021-05-01 um 00 29 27" src="https://user-images.githubusercontent.com/2644614/116760299-e8660500-aa14-11eb-90e6-1b85458c2922.png">

Reply:
Yep, I did my own testing and it's definitely a Big-Sur-or-later feature.

Could you add Note 7 like WebP,
https://github.com/Fyrd/caniuse/blob/13138dde08de5c00dccdd3ad6b4dcb697c7df9c4/features-json/webp.json#L435

And change the statuses like this?
```suggestion
      "14.1":"a #7",
      "TP":"a #7"
```

Reply:
Yep, I did my own testing and it's definitely a Big-Sur-or-later feature.

Could you add a new note 7 somewhere similar to WebP:
> Partial support in Safari refers to being limited to macOS 11.3 or later.

And change the statuses like this?
```
"14.1":"a #4 #5 #7",
"TP":"a #4 #5 #7"
```

Reply:
Yeah, I jumped the gun a bit there on the Safari support, I saw it working on my Big Sur machine and didn't think too hard about it.

Reply:
> Could you add a new note 7 somewhere similar to WebP

Sure!

But are note 4 and 5 still applicable? Or does every video at <https://tests.caniuse.com/webm> just work by default?
Because then it would be more like `a #7` (== partial only because "macOS 11.3 or later"), right?

Reply:
Yeah, I'm fine with removing notes 4 & 5 since those were only worth mentioning when they were the only types of support.

Reply:
There’s one more technicality to think about: VP9 profile 0 and profile 2 in WebRTC. The former is enabled by default in the TP (the note 5) then Safari on macOS Catalina/Mojave would support it too. I’ve been meaning to prepare a PR for this.

I’m not sure if we should track the WebRTC support _at all_ in this feature. Should we simply stop/remove them? This would inevitably make some of Safari’s the WebM support statuses (that rely on the note 4 and/or note 5) to unsupported. The “off-by-default” clause has bothered me anyways.

Reply:
I'm going with `a #7` for now to finally complete this PR, but please follow up @kiding if necessary :)